subprojects/composite-builds/src/test/groovy/org/gradle/composite/internal/DefaultIncludedBuildRegistryTest.groovy

```diff
@@ -243,7 +242,7 @@ class DefaultIncludedBuildRegistryTest extends Specification {
         def nestedBuild = registry.getBuildSrcNestedBuild(rootBuild)
         nestedBuild != null
         nestedBuild.implicitBuild
-        nestedBuild.buildIdentifier == new DefaultBuildIdentifier("buildSrc")
+        nestedBuild.buildIdentifier == new DefaultBuildIdentifier(Path.path(":buildSrc"))
```

Comment:
🤔 Would it make sense to add a factory `buildIdentifier(String)` method to this class?

Reply:
It's a small improvement, but I guess why not 👍